ngraph/src/ngraph/pass/visualize_tree.cpp

```diff
@@ -268,7 +268,8 @@ void pass::VisualizeTree::add_node_arguments(shared_ptr<Node> node,
             auto clone_name = "CLONE_" + to_string(fake_node_ctr);
             auto color = (arg->description() == "Parameter" ? "blue" : "black");
             m_ss << "    " << clone_name << "[shape=\"box\" style=\"dashed,filled\" color=\""
-                 << color << "\" fillcolor=\"white\" label=\"" << get_node_name(arg) << "\"]\n";
+                 << color << "\" fillcolor=\"white\" label=\""
+                 << get_node_name(arg) << "\n" << get_constant_value(arg) << "\"]\n";
```

Comment:
Will it always  print value for any constant? I think it should be parameterized via env variable.